packages/Babylonian-Core.package/BPMethodExample.class/instance/execute.st

```diff
@@ -1,6 +1,11 @@
 tracing
 execute
 	
-	^ self newReceiver 
-		perform: self method selector 
-		withArguments: self newArguments
\ No newline at end of file
+	|executionResult|
+	executionResult := [self newReceiver 
+							perform: self method selector 
+							withArguments: self newArguments]
+								on: Error
```

Comment:
This causes the test failures as it masks the errors we actually want to record. See BlockClosure>>#bpTraceWith: for how this is done. When the code moves there you actually will not need the exception handling in this line anymore.